backend/internal/redmine/models.go

```diff
@@ -75,3 +75,10 @@ type Group struct {
 	Id   int    `json:"id"`
 	Name string `json:"name"`
 }
+type Project struct {
+	TimeEntryActivities []TimeEntryActivity `json:"time_entry_activities"`
+}
+
+type ProjectEntry struct {
+	Project Project `json:"project"`
+}
```

Comment:
I don't really like this type. It seems meaningless to have a struct that _only_ contains another struct. Ideally, this `ProjectEntry` type should disappear and we'd use `Project` instead, but I don't know if that is possible when unmarshalling the project JSON response...

This does not stop me from approving this PR since it actually works and does the right thing.